.github/workflows/release.yaml

```diff
@@ -0,0 +1,73 @@
+name: Create release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag to release (e.g. v1.0.0)"
+        required: true
+        type: string
+      latest:
+        description: "Whether to tag this release latest"
+        required: true
+        type: boolean
+        default: false
+
+jobs:
+  tests:
+    name: Run tests
+    uses: ./.github/workflows/tests.yaml
+
+  verify-tag:
+    name: Verify tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mukunku/tag-exists-action@v1.6.0
+        id: check-tag
+        with:
+          tag: ${{ github.event.inputs.tag }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: fail if tag already exists
+        if: ${{ steps.check-tag.outputs.exists == 'true' }}
+        run: exit 1
+      - name: Validate semver
+        id: parse-semver
+        uses: booxmedialtd/ws-action-parse-semver@v1.4.7
+        with:
+          input_string: ${{ github.event.inputs.tag }}
+          version_extractor_regex: 'v(.*)$'
+
+  release:
+    name: Create a GH release
+    runs-on: ubuntu-latest
+    needs:
+      - tests
+      - verify-tag
+    steps:
+      - name: Parse semver string
+        id: parse-semver
+        uses: booxmedialtd/ws-action-parse-semver@v1.4.7
+        with:
+          input_string: ${{ github.event.inputs.tag }}
+          version_extractor_regex: 'v(.*)$'
```

Comment:
Since we already do this in `verify-tag` and that action has outputs we might as well use that below instead of running this action again. WDYT?

Reply:
Makes sense: 2e3ba05